sub-packages/bionemo-testing/src/bionemo/testing/harnesses/stop_and_go.py

```diff
@@ -217,6 +217,14 @@ def make_callbacks() -> Dict[Type[pl.Callback], pl.Callback]:
                 testing_callbacks.ValidInputCallback: testing_callbacks.ValidInputCallback(),
                 testing_callbacks.ValidOutputCallback: testing_callbacks.ValidOutputCallback(),
                 testing_callbacks.ValidLossCallback: testing_callbacks.ValidLossCallback(),
+                nl_callbacks.ModelCheckpoint: nl_callbacks.ModelCheckpoint(
```

Comment:
moving this here (and instantiating it once for both stop and resume branches) is what's causing that `deepcopy` error.